From foo@baz Tue Apr  9 12:12:43 2002
Date: Mon, 12 Feb 2007 22:33:06 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Driver.h copyright update

It was pointed out that I had not updated my copyright on driver.h

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/device.h |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/include/linux/device.h
+++ gregkh-2.6/include/linux/device.h
@@ -2,6 +2,7 @@
  * device.h - generic, centralized driver model
  *
  * Copyright (c) 2001-2003 Patrick Mochel <mochel@osdl.org>
+ * Copyright (c) 2004-2007 Greg Kroah-Hartman <gregkh@suse.de>
  *
  * This file is released under the GPLv2
  *
